dist/tools/jlink/jlink.sh

```diff
@@ -122,11 +122,20 @@ test_tui() {
     fi
 }
 
+test_opts() {
```

Comment:
`test_opts` could be named `test_serial` I guess. This is more readable and other options should have other test functions imo.